backend/pom.xml

```diff
@@ -16,6 +16,26 @@
     <properties>
         <java.version>11</java.version>
     </properties>
+
+    <profiles>
+        <profile>
+            <id>local</id>
+            <properties>
+                <spring.profiles.active>local</spring.profiles.active>
+            </properties>
+        </profile>
+        <profile>
+            <id>no-tests</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <maven.test.skip>true</maven.test.skip>
+                <spring.profiles.active>no-tests</spring.profiles.active>
```

Comment:
OK I see what is the problem. By default **_no-tests_** maven profile is active if nothing is specified, that means that by default you are loading **_no-tests_** spring profile, so that will search for **_application-no-tests.properties_** which you do not have and parent configuration file will be loaded (application.properties)